tests/fixtures.py

```diff
@@ -118,3 +118,32 @@ def create_pod(request):
     pod_data['metadata']['namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
     pod_data['spec']['volumes'][0]['persistentVolumeClaim']['claimName'] = class_instance.pvc_obj.name
     class_instance.pod_obj = helpers.create_pod(**pod_data)
+
+
+@pytest.fixture()
+def create_project(request):
+    """
+    Create a project
+    """
+    class_instance = request.node.cls
+
+    def finalizer():
+        """
+        Delete the storage class
```

Comment:
```suggestion
        Delete the project
```

Reply:
Done.
Also fixed typo in other places in fixtures.py